auth-web/src/components/auth/account-settings/payment/AccountPaymentMethods.vue

```diff
@@ -45,7 +47,6 @@
     <v-divider class="my-10" />
     <div class="form__btns d-flex">
       <v-btn
-        v-can:CHANGE_PAYMENT_METHOD.disable
```

Comment:
fix for: "the form changes after one save doesn't reset the save button"
disable functionality should handle by  `:disabled="isDisableSaveBtn"`